BUILDING.md

```diff
@@ -19,19 +19,29 @@ You are expected to know your way around with go & git.
 
 If you are a casual user with no development experience, and no preliminary knowledge of go, building from source is probably not a good solution for you.
 
-## Build the development environment
+## Configure the development environment
 
 The first prerequisite of properly building distribution targets is to have a Go
 development environment setup. Please follow [How to Write Go Code](https://golang.org/doc/code.html)
 for proper setup. If done correctly, you should have a GOROOT and GOPATH set in the
 environment.
 
-If a Go development environment is setup, one can use `go get` to install the
-`registry` command from the current latest:
+Next, fetch the code from the repository using git:
 
-    go get github.com/distribution/distribution/cmd/registry
+    git clone git@github.com:distribution/distribution.git
```

Comment:
For docs, I tend to use `https://` instead of `git@`. Doing so allows users to clone without any setup (so avoids us having to explain how to set things up);

```bash
git clone https://github.com/distribution/distribution.git
```

It's also easier, as it allows them to copy the repository URL from their browser 😅

Reply:
With our without the .git? They both work, but I expect you mean without.

Reply:
I think the git suffix saves an extra lookup, but either should work